lib/runtime/src/lib.rs

```diff
@@ -100,12 +98,26 @@ pub mod wasm {
     pub use wasmer_runtime_core::types::{FuncSig, MemoryDescriptor, TableDescriptor, Type, Value};
 }
 
+pub mod error {
+    #[cfg(feature = "default-compiler")]
+    pub use super::cache::Error as CacheError;
+    pub use wasmer_runtime_core::error::*;
+}
+
 pub mod units {
     //! Various unit types.
-
     pub use wasmer_runtime_core::units::{Bytes, Pages};
 }
 
+#[cfg(feature = "default-compiler")]
+mod cache;
+
+#[cfg(feature = "default-compiler")]
+use wasmer_runtime_core::backend::Compiler;
+
+#[cfg(feature = "default-compiler")]
```

Comment:
Shouldn't this be `cache` ?

Reply:
cache relies on there being a default-compiler.